### ISSUE TITLE
Profile picture changing with copied image

### ISSUE THREAD

**Issue by Thingyman-DEV** [LLK/scratch-www#4468]
_If you find an image link, then copy the image, then paste it in a comment on your profile, not the link, but the image itself, then your profile picture will change to that image_

### Steps to Reproduce

_Go to an image [link](https://en.scratch-wiki.info/w/images/thumb/ScratchCat-Small.png/200px-ScratchCat-Small.png), then right click and click copy image, then go to your profile. Then in a new comment, paste the image. Nothing will show up, but on the top of the screen, you will get the usual notification for changing a profile pic(and the one saying it's too big if it's too big)_

### Operating System and Browser

_e.g. Mac OS 10.15.7 Using google chrome, however za-chary can also confirm that it works on his phone_

**Comment by AmazingMech2418** [comment]
https://scratch.mit.edu/discuss/topic/443284/ has some discussion on this specific bug. Personally, I think it could be a great feature to allow copying and pasting, but in a designated spot rather than anywhere on the profile. Also, it does not happen just with comments, but also with anywhere on the page.

**Comment by JeffaloBob** [comment]
this is a scratchr2 bug

**Comment by AmazingMech2418** [comment]
@JeffaloBob Yeah, but it's a private repo. The most we can see is the client-side just using DevTools.

**Comment by AmazingMech2418** [comment]
jQuery seems to be a part of this. It appears that the uploading system is through jQuery.

**Comment by SausageMcSauce** [comment]
(Extending on what JeffaloBob replied) Profile pages are not part of this repository, instead they are in a private one. According to other comments on issues, it is recommended to discuss this on the Bugs and Glitches topic mentioned above, instead of here.
